pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.80</version>
+    <version>4.81</version>
```

Comment:
Isn't it risking conflict with the automatic dependency PR  https://github.com/jenkinsci/bitbucket-branch-source-plugin/pull/844 ?

Reply:
if so i will resolve it